Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -131,6 +131,7 @@ screen initialState =
                   launchAff_ $ flowRunner $ getQuotesPolling (getValueToLocalStore TRACKING_ID) GetQuotesList Restart pollingCount (fromMaybe 0.0 (NUM.fromString (getValueToLocalStore TEST_POLLING_INTERVAL))) push initialState
               ConfirmingRide -> launchAff_ $ flowRunner $ confirmRide GetRideConfirmation 5 1000.0 push initialState
               HomeScreen -> do
+                _ <- pure $ setValueToLocalStore SESSION_ID (generateSessionId unit)
```

Comment:
do we want same session id for a single ride ? if yes then can we call it where we are searching for ride